src/helpers/mod.rs

```diff
@@ -27,6 +27,15 @@ use std::ops::Index;
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
+/// Represents a unique identity of each helper running MPC computation.
+#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "enable-serde", derive(Serialize, Deserialize))]
+pub enum Identity {
```

Comment:
Maybe for later, but it would be good to have functions either like this:

```rust
fn left(self) -> Self { ... }
fn right(self) -> Self { ... }
```
or like this:

```rust
fn peer(self, dir: Direction) -> Self { ... }
```

where:

```rust
#[derive(Copy, Clone, Debug, PartialEq, Eq)]
enum Direction { Left, Right }
```

Reply:
might be good to add the `left()`/`right()` at same place as `identity()`, on `Ring`

Reply:
hmm, I kinda like this interface now

```rust
   async fn send<T>(peer: Identity, msg: T)
```

and calling it from multiplication like this

```rust
   ring.send(ctx.identity.peer(Direction::Right), d_value)
```